napari/layers/_tests/test_dask_layers.py

```diff
@@ -29,7 +29,7 @@ def get_plane(block_id):
     )
     layer = layers.Image(arr)
     assert FETCH_COUNT == 1
-    assert tuple(layer.contrast_limits) == (0, 1)
+    assert tuple(layer.contrast_limits) != (0, 1)
```

Comment:
Can you add a comment here on why this is now `!=`. Also could you "trigger" a "set_view_slice" and then get the `==`? Maybe i'm really not quite getting what's going on in test if that question makes zero sense - it's also late right now ... ;-)

Reply:
(0, 1) was the "fallback" contrast limits when we didn't calculate.  now I'm simply asserting that we _have_ actually set the contrast limits to something (but it's a random array, so I didn't try to assert the actual number)

Reply:
Adding that comment would help. +1 on a comment :)

Reply:
added a more explicit/self-evident test that actually checks it's equal to min/max of the array